examples/step-2/doc/results.dox

```diff
@@ -38,11 +12,11 @@ equation):
 <TABLE WIDTH="60%" ALIGN="center">
   <tr>
     <td ALIGN="center">
-      <img src="http://www.dealii.org/images/steps/developer/step-2.sparsity-1.png" alt="">
+      @image html sparsity_pattern1.svg
```

Comment:
These will eventually have to move to the server. Please use `http://www.dealii.org/images/steps/developer/step-2.sparsity-1.svg` and similarly for the second one below.